src/server/cloak-server.js

```diff
@@ -162,6 +177,49 @@ function setNextLeader(room) {
     room.data.leaderIndex = nextLeader;
 }
 
-function gameController(room) {
-    var roomTimer = setInterval(setNextLeader.bind(null, room), 5000);
+function checkListLength(user) {
+    var room = user.getRoom();
+    var letterList = room.data.letterList;
+    if(letterList.letters.length >= 9){
+        letterList.disableConsonant = true;
+        letterList.disableVowel = true;
+        user.message('disableConsonant', true);
+        user.message('disableVowel', true);
+        return;
+    }
+}
+
+function getConsonant(arg, user) {
```

Comment:
Is this or anything new being tested? I know the tests have been updated, but those are for other things right?